app/uk/gov/hmrc/gform/formtemplate/FormTemplateValidator.scala

```diff
@@ -784,9 +784,14 @@ object FormTemplateValidator {
           internalSmartString.rawValue(LangADT.En).trim =!= "" && internalSmartString.rawValue(LangADT.Cy).trim === ""
         )
 
+    def ignoredComponent(fc: FormComponent) = fc match {
+      case IsInformationMessage(_) | IsMiniSummaryList(_) | IsTableComp(_) => true
+      case _                                                               => false
+    }
+
     val validationResults = sectionsList.flatMap { page =>
       (page.allFormComponents ++ page.fields).collect {
-        case fc if isEmptyLabel(fc.label) && isEmptyCy(page.title) =>
+        case fc if !ignoredComponent(fc) && isEmptyLabel(fc.label) && isEmptyCy(page.title) =>
```

Comment:
Why this branch even exists? Why are we checking empty welsh label?

Reply:
We should check the empty welsh label according to AC1 in [GFORMS-2885](https://jira.tools.tax.service.gov.uk/browse/GFORMS-2885). It does not allow us to post a form if section or page with a title of "cy": "" where the label is not present

Reply:
I remember I've posted this: https://github.com/hmrc/gform/pull/1212#pullrequestreview-2303534860
was there some reason why this wasn't implemented and AC1 dropped?